main.tf

```diff
@@ -48,6 +54,21 @@ locals {
 
 ##############################################################################
 
+##############################################################################
```

Comment:
The VPC should not be created in the root module - the root module would focus only on the VPE aspect. The VPC can be created in the examples as needed